tests/security/selinux/container_selinux.pm

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: test that containers are using SELinux
+# Maintainer: QE Security <none@suse.de>
+# Tags: poo#119827
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    # ensure that SELinux is enabled and in enforcing mode
+    validate_script_output("sestatus", sub { m/SELinux\ status: .*enabled.* Current\ mode: .*enforcing/sx });
+
+    my $image = 'registry.opensuse.org/opensuse/tumbleweed:latest';
+    my $container_name = 'test_container_selinux';
+
+    assert_script_run("podman pull $image");
+    assert_script_run("podman run --name $container_name -dt $image");
```

Comment:
`podman run` on a new image implicitly does a pull, so maybe it's not necessary ?
since images can be quite big, I'd insert a generous timeout here :)

Reply:
I'd prefer to keep those cmds separate, since once it's involving downloading and the other running a container.
Tumbleweed image is about 40MB, so 90s should always be more than enough. But I agree that it's better to be safe than sorry :) I'll put 300s.